docs/chapter4-payload-handoff-format.rst

```diff
@@ -429,7 +429,7 @@ which configuration was chosen. It cannot find this out by itself. The
 'image' node provides information to help with these problems.
 
 .. tabularcolumns:: | p{4cm} p{0.75cm} p{4cm} p{6.5cm} |
-.. table:: Node: /options/upl-images@<addr>
+.. table:: Node: /options/upl-image@<addr>
```

Comment:
from previous google doc, upl-images is the right one, why we wanted to rename it?
https://docs.google.com/document/d/1WxEUlCsXpc17DkJhL3XVkOW7e_KIt_zcc9tQ1trOySg/edit?pli=1&tab=t.0#heading=h.l4fyrh27u9hh

Reply:
my thought is, if we changed to upl-images, then it also make sense to change the other node to upl-images too:
https://universalpayload.github.io/spec/chapter4-payload-handoff-format.html#node-options-upl-image-image-addr-o

Then will have to change 2 parts instead of 1. what do you think?

Reply:
Just my 2 cents,
"upl-images" is the whole UPL binary information which may combine separate child images together for different configuration/feature support scenarios. In this case "images" looks better.
"upl-images/image" is single image information so it should be "image" (no s)

Reply:
Per meeting, update it to align with the name same as the title aabove.

Reply:
with discussion we are ok to have image here.